### ISSUE TITLE
seq2seq multidimensional regression

### ISSUE THREAD

**Issue by guillaume-chevalier** [tensorflow/tensorflow#3487]
Hi, I am facing an issue while trying to train an RNN to do a multidimensional regression.  

**The Tensorflow loss function built for RNNs seems to address the cases where we directly want to train labels or words embeddings, so I tried to compute the loss myself for a regression and it failed or seems not done for that.** 

The question is also more detailed with my actual code and Tensorflow Graph on [StackOverflow](http://stackoverflow.com/questions/38549040/tensorflow-seq2seq-multidimensional-regression).
_This seems more like a missing feature, so this is why I ask the question here too._ 

What would be a nice workaround? There **should** be one.

If there is already code for that, I would really like to see it, because what I want to do seems undocumented yet and I would like to do it, I am probably not alone. In the end, I would like to predict the N next coeficients of some [STFT](https://www.youtube.com/watch?v=TZzS52OplYs&index=5&list=PLlp-GWNOd6m6gSz0wIcpvl4ixSlS-HEmr), but for now I focus on making the regression model with simple sinusoidal data. 

Many thanks!!!

**Comment by concretevitamin** [comment]
@lukaszkaiser: can you take a look?  Thanks.

**Comment by denlecoeuche** [comment]
@guillaume-chevalier Are you sure about using `tf.nn.softmax_cross_entropy_with_logits` ? If I'm not mistaken MSE is usually more suited for regression problems.

**Comment by lukaszkaiser** [comment]
This is a general request for information, not a bug or concrete feature request, so indeed - Stack Overflow is a better place for that. I'll answer there, closing this issue.

**Comment by guillaume-chevalier** [comment]
Many thanks for the help, using an MSE fixed the problem, and using L2 loss improved the result too.
